recipe/meta.yaml

```diff
@@ -17,6 +17,10 @@ build:
 
 requirements:
   build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
+    - vtk                                    # [build_platform != target_platform]
```

Comment:
@hoechenberger I added `vtk` here because I noticed the bot added `numpy`. I suspect this might be what's needed, we'll see!

Reply:
Good idea!